test/blocks/simplified-pricing-cards/simplified-pricing-cards.test.js

```diff
@@ -0,0 +1,57 @@
+import sinon from 'sinon';
+import { readFile } from '@web/test-runner-commands';
+import { expect } from '@esm-bundle/chai';
+import { mockRes } from '../test-utilities.js';
+
+// const locales = { '': { ietf: 'en-US', tk: 'hah7vzn.css' } };
+// const { getLibs } = await import('../../../express/scripts/utils.js');
+// await import(`${getLibs()}/utils/utils.js`).then((mod) => {
+//   const conf = { locales };
+//   mod.setConfig(conf);
+// });
+// eslint-disable-next-line import/no-unresolved
+const { default: decorate } = await import('../../../../express/blocks/simplified-pricing-cards/simplified-pricing-cards.js');
```

Comment:
hint: your tests are failing because you haven't initialized your config correctly. To do so, take a look at how we're doing the importing for example in `how-to-cards.test.js`. That should fix your tests.

Reply:
I had to change this command in package.json to get the tests to run as well.

 "wtr \"./test/**/*.nothing.test.js\" --node-resolve --port=2000 --coverage",

Reply:
yes, that change was a small test I did on main, sorry for that.